go/ai-proxy/proxy/bedrock/bedrock.go

```diff
@@ -0,0 +1,359 @@
+package bedrock
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"reflect"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/bedrockruntime"
+	"github.com/aws/aws-sdk-go-v2/service/bedrockruntime/document"
+	"github.com/aws/aws-sdk-go-v2/service/bedrockruntime/types"
+	"github.com/google/uuid"
+	ollamaapi "github.com/ollama/ollama/api"
+	"github.com/ollama/ollama/openai"
+	"k8s.io/klog/v2"
+
+	"github.com/pluralsh/console/go/ai-proxy/api"
+)
+
+const (
+	ChatCompletionObject      = "chat.completion"
+	ChatCompletionChunkObject = "chat.completion.chunk"
+)
+
+type BedrockProxy struct {
+	region        string // might not be necessary but check
```

Comment:
I believe this can be inferred, so we should make it optional config, and don't add it to the opts if not present

Reply:
eks IRSA for instance auto-adds the `AWS_REGION` env var